docs/specification/current.md

```diff
@@ -31,28 +32,28 @@ decision support from within a clinician's workflow. The API supports:
 * Launching a web page to provide additional information to the user
 * Launching a user-facing SMART app when CDS requires additional interaction
 
-The basic components of CDS Hooks are:
+The main concepts of the specification are Services, CDS Clients, and Cards.
 
 ### CDS Services
-In CDS Hooks, a _CDS Service_ is a service that provides recommendations and guidance through RESTful APIs as described by this specification. The primary APIs are [Discovery](#discovery), which allows a CDS Developer to publish the types of CDS Services it provides, the [Service](#calling-a-cds-service) endpoint that CDS Clients use to request decision support, and the [Feedback](#feedback) service through which services learn the outcomes of their recommendations and guidance.
+A _CDS Service_ is a service that provides recommendations and guidance through the RESTful APIs described by this specification. The primary APIs are [Discovery](#discovery), which allows a CDS Developer to publish the types of CDS Services it provides. The [Service](#calling-a-cds-service) API that CDS Clients use to request decision support. The  [Feedback](#feedback) API through which services learn the outcomes of their recommendations and guidance.
 
 ### CDS Clients
-A _CDS Client_ is an electronic health record, or other clinical information system that consumes decision support by calling CDS Services at specific points in the application's workflow called [_hooks_](#hooks). Each hook defines the _hook context_, contextual information available within the CDS Client and specific to the workflow and provided as part of the request. Each service advertises which hooks it supports and what [_prefetch data_](#providing-fhir-resources-to-a-cds-service) (information needed by the CDS Service to determine what decision support should be presented) it requires. In addition, CDS Clients may provide an authorization and FHIR resource server as part of the request to enable services to request additional information.
+A _CDS Client_ is an Electronic Health Record (EHR), or other clinical information system that uses decision support by calling CDS Services at specific points in the application's workflow called [_hooks_](#hooks). Each hook defines the _hook context_ (contextual information available within the CDS Client and specific to the workflow) that is provided as part of the request. Each service advertises which hooks it supports and what [_prefetch data_](#providing-fhir-resources-to-a-cds-service) (information needed by the CDS Service to determine what decision support should be presented) it requires. In addition, CDS Clients typically provide the FHIR resource server location and associated authorization information as part of the request to enable services to request additional information.
```

Comment:
```suggestion
A _CDS Client_ is an Electronic Health Record (EHR), or other clinical information system that consumes decision support by calling CDS Services at specific points in the application's workflow called [_hooks_](#hooks). Each hook defines the _hook context_ (contextual information available within the CDS Client and specific to the workflow) that is provided as part of the request. Each service advertises which hooks it supports and what [_prefetch data_](#providing-fhir-resources-to-a-cds-service) (information needed by the CDS Service to determine what decision support should be presented) it requires. In addition, CDS Clients typically provide the FHIR resource server location and associated authorization information as part of the request to enable services to request additional information.
```
I think that consumes is both more descriptive and therefore accessible to readers, but also more accurate in that it doesn't imply that the cds provided by the service is acted upon / "used".

If you really don't like the word consumed, "accessed" would be a good middle ground.

Reply:
I think, I prefer the word accesses over consumes.

Reply:
I agree with @bvdh. 'consumes' just feels a bit weird. I know people are pushing for "consuming an API" and this is a derivation of that, but it's still weird and would be a bit harder than necessary for non-native English speakers.